### DISCUSSION TITLE
I have Hamgeek HMG-01 Plus POE device, but have choosen UZG-01 during initial setup

### DISCUSSION THREAD

**Discussion by vit5041** [xyzroe/XZG#127] [General]
hi All,
I've just figured out that my model in WebUI is incorrect. But it seems all works fine and HMG-01 is exact clone of UZG-01..
But still have doubt, what kind of issues I may face, should I change the model or can just leave it.  is it possible to change it without issues / loosing any settings. 
Thanks a lot!

**Comment by xyzroe** [comment]
No issues. This is just a name. 
The only way to change - factory reset the device.

**Reply by vit5041** [reply]
Thanks a lot for the confirmation!
